app/assets/stylesheets/admin/general.scss

```diff
@@ -22,3 +22,44 @@ table.key-value {
   flex-flow: row wrap;
   justify-content: space-between;
 }
+
+.right-align {
+  text-align: right;
+}
+
+.left-box, .right-box {
```

Comment:
Looks like neither of these are used anywhere. I am also not a fan of these non-semantic classes, but I know there are different schools of thought on this. We should probably choose one and stick to it. Let's discuss. But for now since these are not used, please remove?